src/libmain.c

```diff
@@ -462,6 +462,18 @@ gboolean main_is_realized(void)
 }
 
 
+/**
+ *  Checks whether Geany is 'closing all' tabs right now.
+ *
+ *  @return @c TRUE if the Geany is 'closing all' tabs right now or @c FALSE otherwise.
+ **/
+GEANY_API_SYMBOL
+gboolean geany_is_closing_all_tabs(void)
```

Comment:
I'd probably call it `geany_is_closing_all_documents()`.

Reply:
renamed